collectors/slabinfo.plugin/README.md

```diff
@@ -0,0 +1,25 @@
+# slabinfo.plugin
+
+SLAB is a cache mechanism used by the Kernel to avoid fragmentation.
+
+Each internal structure (process, file descriptor, inode...) is stored within a SLAB.
+
+
+## configuring Netdata for slabinfo
+
+There is currently no configuration needed, aside to have the `slabinfo.plugin` plugin being able to read 
+`/proc/slabinfo`. As this procfile is only readable by root, (mode `0400` in the kernel source), we need to grant the 
+slabinfo.plugin this access, either by:
+-  with Setuid root: `chown root: slabinfo.plugin && chmod u+s slabinfo.plugin`
+-  with capability dac_override: `setcap cap_dac_override+ep slabinfo.plugin`
```

Comment:
You've already granted access to the `slabinfo` file at installation time. I don't think users need to know about other possible ways of configuring it. You can use a simple sentence, something like
> Keep in mind that libxenstat requires root access, so the plugin is setuid to root.

Reply:
Is that better ? 

> There is currently no configuration needed.
>
> As `/proc/slabinfo` is only readable by root, this plugin needs to be setuid root.

Reply:
LGTM